### ISSUE TITLE
Rust Analyer can't recognize the IUnknown

### ISSUE THREAD

**Issue by Ciantic** [microsoft/com-rs#195]
It's very hard to develop on 0.3 since [Rust Analyzer](https://github.com/rust-analyzer/rust-analyzer) can't recognize the IUnknown.

In VSCode settings.json I have this:

```
{
    "rust-analyzer.procMacro.enable": true
}
```

yet if I write this:

```
use com::{interfaces, interfaces::iunknown::IUnknown};

fn main() {
    println!("Hello, world!"); 
}
```

It gives unresolved import:

![image](https://user-images.githubusercontent.com/64731/103642240-8b24e000-4f5b-11eb-988a-c277d447fb21.png)

Yet it builds just fine with `cargo build`.

I think RLS support is a must, developing without IDE support is pretty difficult, maybe I have misconfigured my setup somehow. 

Also from the get-go, I think I don't like the macro approach in 0.3, because I suspect it will break Go to definition etc. IDE features that used to work with trait approach.

**Comment by Ciantic** [comment]
I've also raised this in Rust Analyzer forums: https://users.rust-lang.org/t/proc-macros-does-not-work-with-com-crate/53731

I can't understand how one can program without intellisense :) I need to get this working.

**Comment by MarijnS95** [comment]
@ciantic Afaik this has been working fine in the past, but even going back to `rust-analyzer` 2020-10-05 shows the same issue. Oddly enough jump-to-symbol works in the `webview2` example.

Note that `IUnknown` is a plain struct here that's not wrapped/generated by proc-macro...

**Comment by rylev** [comment]
This was also raised [here on Zulip](https://rust-lang.zulipchat.com/#narrow/stream/122651-general/topic/proc_macro.20support). You'll need to enable both "Load out dirs from check" and "Enable proc macro support".

**Comment by Ciantic** [comment]
Uh, looks like my Zulip thing has gone forward more, someone opened an issue https://github.com/rust-analyzer/rust-analyzer/pull/7171

**Comment by MarijnS95** [comment]
> **Edit** I see you managed to repro this with all the way to 2020-10-05? Well something is broken in analyzer in that case.

@Ciantic  Yup. Could have been (way) longer since I last actively developed on a crate with `com-rs` though.

Rebuilt RA at https://github.com/rust-analyzer/rust-analyzer/commit/861a547 (includes that PR), doesn't solve it (don't think it was intended to, but I have not read into the PR/Zulip yet).
